target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -442,8 +445,23 @@
 			compatible = "pci168c,0046";
 			reg = <0x00010000 0 0 0 0>;
 
-			mtd-mac-address = <&art 6>;
+			nvmem-cells = <&macaddr_art_6>;
+			nvmem-cell-names = "mac-address";
 			mac-address-increment = <(2)>;
 		};
 	};
 };
+
+&art {
+	compatible = "nvmem-cells";
+	#address-cells = <1>;
+	#size-cells = <1>;
```

Comment:
Being a child node of the partitions that have the same setup, wouldn't the address-/size-cells be inherited, so we could omit them here (in general)?

Reply:
I need to check but i remember that the buildroot complained with missing address and size without this